README.md

```diff
@@ -20,3 +20,23 @@ Authentication:
 
 - [Federal Central Invoice Submission Portal (ZRE)](https://xrechnung.bund.de/prod/authenticate.do)
 - [Online Access Act-compliant Invoice Submission Portal](https://xrechnung-bdr.de/edi/auth/login)
+
+---
+
+## Requirements
+
+Required libraries in your system:
```

Comment:
I'd assume these are testing requirements?

Reply:
Yes. We can call it more specifically.